zomes/blog/code/src/lib.rs

```diff
@@ -1,12 +1,25 @@
 #[macro_use]
 extern crate hdk;
-extern crate holochain_wasm_utils;
 extern crate serde;
 #[macro_use]
 extern crate serde_derive;
 #[macro_use]
 extern crate serde_json;
+extern crate boolinator;
 
 pub mod hc_lifecycle_capability;
 pub mod main_capability;
-pub mod post_validations;
+pub mod post;
+
+use hdk::meta::ZomeDefinition;
+
+/// This function needs to be implemented by every zome.
+/// Every entry type needs to be registered by passing its
+/// definition object (constructed by entry!) to the define
+/// function.
+///
+/// Will soon be replaced by a define_zome! macro
+#[no_mangle]
+pub extern fn zome_setup(zd: &mut ZomeDefinition) {
```

Comment:
I think zd could be expanded, so that it's clearer. Like `zome_defn`

Reply:
Yes